pytorch_lightning/utilities/imports.py

```diff
@@ -109,6 +109,9 @@ def _compare_version(package: str, op: Callable, version: str, use_base_version:
 else:
     _IPU_AVAILABLE = False
 
+from habana_frameworks.torch.utils.library_loader import is_habana_avaialble
+
+_HPU_AVAILABLE = is_habana_avaialble()
```

Comment:
```suggestion
try:
    from habana_frameworks.torch.utils.library_loader import is_habana_avaialble

    _HPU_AVAILABLE = is_habana_avaialble()
except ImportError:
    _HPU_AVAILABLE = False
```

Reply:
Here, we could check availability of two things

1. Habana framework
2. Habana Devices

You could rather check this via

```python

_HABANA_AVAILABLE = _module_available("habana")
if _HABANA_AVAILABLE:
	from habana_frameworks.torch.utils.library_loader import is_habana_available
	
	_HPU_AVAILABLE = is_habana_avaialble()
else:
	_HPU_AVAILABLE = False
```

Reply:
Am already doing it  within the function.